apps/web/components/v2/eventtype/EventAdvancedTab.tsx

```diff
@@ -0,0 +1,202 @@
+import { EventTypeSetupInfered, FormValues } from "pages/v2/event-types/[type]";
```

Comment:
These set of components in V2 need to stay in here - mainly for using inferred types making this a lot easier and more future proof.